packages/x-date-pickers/src/internals/hooks/useStaticPicker/useStaticPicker.tsx

```diff
@@ -30,7 +30,15 @@ export const useStaticPicker = <
   validator,
   ref,
 }: UseStaticPickerParams<TDate, TView, TExternalProps>) => {
-  const { localeText, components, componentsProps, displayStaticWrapperAs, autoFocus } = props;
+  const {
+    localeText,
+    slots,
+    slotsProps,
+    components,
+    componentsProps,
```

Comment:
How come for `useMobilePicker` we can only keep `slots` and `slotsProps`, but for `useStaticPicker` we are leaving these props as well? 🤔 
Is it not a mistake by chance?

Reply:
Yes, it sounds to be a mistake